docs/src/main/sphinx/admin/fault-tolerant-execution.rst

```diff
@@ -280,6 +301,19 @@ fault-tolerant execution. You can configure a filesystem-based exchange manager
 that stores spooled data in a specified location, either an S3-compatible
 storage system or a local filesystem.
 
+Requirements
+^^^^^^^^^^^^
+
+Exchange spooling with an exchange manager requires the following:
+
+* The ``query.hash-partition-count`` :doc:`query management property
+  </admin/properties-query-management>` must be set to a value of ``50`` or
+  lower in ``config.properties``, as the filesystem exchanage manager supports
```

Comment:
Drop the "in config.properties"  .. its redundant

Reply:
I'm thinking about introducing a dedicated property for fault tolerant execution with an appropriate default. The discussion is happening here: https://github.com/trinodb/trino/pull/12228#issuecomment-1117673220

Reply:
Will keep an eye on that and make edits based on how that PR resolves